pcsx2/gui/DriveList.cpp

```diff
@@ -0,0 +1,97 @@
+#include "PrecompiledHeader.h"
```

Comment:
Forgot to include copyright header in this file.

Reply:
I knew I was forgetting something